SETUP.md

```diff
@@ -171,18 +171,30 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
 Note: Virtual Machine Users should check the [Alternative note](#alternative-use-an-ubuntu-vm) below before starting
 
 1. `sudo apt-get update`
-1. `sudo apt-get install -y git mysql-server mysql-client libmysqlclient-dev libxslt1-dev libssl-dev zlib1g-dev imagemagick libmagickcore-dev libmagickwand-dev openjdk-9-jre-headless libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev curl pdftk enscript libsqlite3-dev build-essential redis-server rbenv chromium-browser parallel`
+1. `sudo apt-get install -y git mysql-server mysql-client libmysqlclient-dev libxslt1-dev libssl-dev zlib1g-dev imagemagick libmagickcore-dev libmagickwand-dev openjdk-11-jre-headless libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev curl enscript libsqlite3-dev build-essential redis-server rbenv chromium-browser parallel`
     * **Hit enter and select default options for any configuration popups, leaving mysql passwords blank**
 1. *(If working from an EC2 instance)* `sudo apt-get install -y libreadline-dev libffi-dev`
+1. configure your system so that `~/.bashrc` (or another startup file of your choice) will be run whenever you open a shell
+    1. if you are using bash and setting up a new linux system, follow [this explanation](https://joshstaiger.org/archives/2005/07/bash_profile_vs.html) and add the following lines to your `~/.bash_profile` config file:
+
+        ```
+        if [ -f ~/.bashrc ]; then
+          source ~/.bashrc
+        fi     
+        ```
+
```

Comment:
note that newer systems use `.profile` rather than `.bash_profile`, and that often it's not necessary to set this up manually. What about something like 

```suggestion
1. configure your system so that `~/.bashrc` (or another startup file of your choice) will be run whenever you open a shell
    1. if you are using bash and setting up a new linux system, you may need to modify `~/.bash_profile` or `~/.profile` (your login shell configuration file) as per [this explanation](https://joshstaiger.org/archives/2005/07/bash_profile_vs.html), which recommends adding these lines:

        ```
        if [ -f ~/.bashrc ]; then
          source ~/.bashrc
        fi     
        ```
```